test/unittests/UserTest.php

```diff
@@ -58,7 +58,7 @@ class UserTest extends TestCase
                 'DBAccess'               => '123',
                 'Active'                 => 'Y',
                 'Password_hash'          => null,
-                'Password_expiry'        => '2020-07-16',
+                'PasswordChangeRequired'        => 0,
```

Comment:
not sure if this needs phpcs to pass travis

Reply:
Me neither!